src/main/java/com/audiolib/persistance/model/Artist.java

```diff
@@ -2,24 +2,29 @@
 
 import java.util.Set;
 
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.OneToMany;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
 @Entity
 public class Artist {
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name="ArtistId")
     private Long id;
 
-    @Column(name = "Name")
+    @Column(name = "Name", nullable = false) // nullable permet de gerer les erreur si le title est vide
     private String name;
 
-    @OneToMany(mappedBy = "artist")
+    @JsonIgnoreProperties("artist")
+    @OneToMany(mappedBy = "artist", fetch=FetchType.EAGER, cascade = CascadeType.ALL)
 	private Set<Album> albums;
```

Comment:
Il est préférable d'initialiser les `HashSet` pour pouvoir ajouter des éléments dedans sans se préoccuper de savoir s'il est null
```suggestion
	private Set<Album> albums = new HashSet<>();
```
Qui dit HashSet dit hashCode... Du coup le hashCode au niveau d'albums est important (même si tu gères l'unicité dans la partie contrôleur)

Reply:
Idem pour les equals, toString et hashCode ici